.changes/next-release/feature-AWSSDKforJavav2-418a15a.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "AWS SDK for Java v2", 
```

Comment:
Since this is specific to Apache, we should change this to the Apache client

Reply:
ok.